ZarCare Automation/Test/PageActions/FamilyMember_Page.cs

```diff
@@ -80,55 +80,177 @@ public static void AddfamilyMemberDetail(string MemFirstName, string MemLastName
+            Wait.WaitTillPageLoad();
+
+
+            int MemberCount = FamilyMemberPage.Web_FamilyMemberName.Count;
+
+            for (int i = 0; i < MemberCount; i++)
+            {
+                string MemberName = FamilyMemberPage.Web_FamilyMemberName[i].Text;
+
+                if (MemberName.Contains(AddedMemFirstName))
+                {
+                    string AddedMemeberFullName = FamilyMemberPage.Web_FamilyMemberName[i].Text;
+
+                    string[] AddedMemFirstNameSplit = AddedMemeberFullName.Split(' ');
+                    string AddedMember = AddedMemFirstNameSplit[0];
+
                     
+                    Assert.That(AddedMemFirstName, Is.EqualTo(AddedMember));
+
+                    // Relationship text validation
+
+                    string RelationShipFullText = FamilyMemberPage.Web_RelationshipText[i].Text;
+
                     bool isTextMatch = RelationShipFullText.Contains(MemRelationshipText, StringComparison.OrdinalIgnoreCase);
 
                     // Assert to validate  realtionship text
-                    Assert.IsTrue(isTextMatch, $"The text '{RelationShipFullText}' does not contain '{MemRelationshipText}");
+                    StringAssert.Contains(MemRelationshipText, RelationShipFullText, $"The text '{RelationShipFullText}' does not contain '{MemRelationshipText}'");                 
                     break;
-
                 }
-                else
+            }
+            Generic_Utils.ScrollToBottom();
+            Wait.WaitTillPageLoad();
+            Reports.childLog.Log(Status.Info, "Added Member display in medical file");
+            Generic_Utils.GetScreenshot("Added Member display in medical file screenshot");
+
+        }
+        public static void AddedmemberAppearInMemberDropdown(string appointment_Time, string MemAndRelation)
+        {
+            FamilyMemberPage.Web_BookOppntTab.Click();
+            Wait.WaitTillPageLoad();
+            FamilyMemberPage.Web_BookopntBtn.Click();
+            Wait.WaitTillPageLoad();
+
+            int slot_count = DoctorProfilePage.Web_BookingSlot_Currentday.Count;
+
+            for (int a = 0; a < slot_count; a++)
+            {
+                string slot_time = DoctorProfilePage.Web_BookingSlot_Currentday[a].Text;
+
+                if (slot_time.Equals(appointment_Time))
                 {
-                    Console.WriteLine("No member name found into list");
+                    DoctorProfilePage.Web_BookingSlot_Currentday[a].Click();
+                    break;
                 }
             }
-        }
+            Wait.WaitTillPageLoad();
+            Reports.childLog.Log(Status.Info, "Book Oppointment page displayed");
+            Generic_Utils.GetScreenshot("Book Oppointment page displayed screenshot");
+
+            FamilyMemberPage.Web_AddMemebrInfoCheckBtn.Click();
 
+            SelectElement selectElement = new SelectElement(FamilyMemberPage.Web_DropDownMember);
```

Comment:
Please replace this code with generic dropdown method

Reply:
This is not Select drop down element it is like verify drop down element is present or not from family member drop down